css/style.css

```diff
@@ -149,3 +149,555 @@ section .content p {
     margin-left: 45px;
   }
 }
+
+.section-1 .first-header h2 {
+  position: static;
+  width: 327px;
+  height: 97px;
+  font-size: 40px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+  margin-top: 60rem;
```

Comment:
- [ ] There is less `margin-top` in original design. Please try to fix this issue.

![image](https://user-images.githubusercontent.com/53019916/125810342-c57e4190-2d55-4534-a4a0-96f043689c55.png)